shift_detector/precalculations/TextMetadata.py

```diff
@@ -427,6 +428,30 @@ def metadata_function(self, text):
         return textstat.text_standard(text, True)
 
 
+class PartOfSpeechMetadata(GenericTextMetadata):
+
+    @staticmethod
+    def metadata_name() -> str:
+        return 'pos_tags'
+
+    def metadata_return_type(self) -> ColumnType:
+        return ColumnType.categorical
+
+    @staticmethod
+    def tag_histogram(text):
+        tokenized_text = nltk.word_tokenize(text)
```

Comment:
it tokenizes in another way then our tokenizing. nltk makes punctuation into separate tokens, we eliminate the punctuation because our metrics would be wrong if every punctuation counted as it's own word. 
Part of speech needs the punctuation, so we have to use the nltk tokenization here